src/samples/ChildBenefitsClaim/index.tsx

```diff
@@ -131,12 +131,23 @@ export default function ChildBenefitsClaim() {
 
   const [inprogressClaims, setInprogressClaims] = useState([]);
   const [submittedClaims, setSubmittedClaims] = useState([]);
+  const [showBeginNewClaimButton, setShowBeginNewClaimButton] = useState(false);
 
   function doRedirectDone() {
     history.replace('/');
     // appName and mainRedirect params have to be same as earlier invocation
     loginIfNecessary({ appName: 'embedded', mainRedirect: true });
   }
+  useEffect(() => {
+    if (submittedClaims[0]?.HideBeginClaimSubmitted === true) {
+      setShowBeginNewClaimButton(true);
+    } else if (
+      submittedClaims[0]?.HideBeginClaimSubmitted === false ||
+      submittedClaims[0]?.HideBeginClaimSubmitted === null
+    ) {
+      setShowBeginNewClaimButton(false);
+    }
```

Comment:
We could probably just do an else here because null and false are both false

Reply:
added